astroquery/esasky/tests/test_esasky_remote.py

```diff
@@ -24,6 +25,7 @@ def test_esasky_query_object_maps(self):
         result = ESASkyClass.query_object_maps("M51")
         assert isinstance(result, TableList)
 
+    @pytest.mark.bigdata
```

Comment:
out of curiosity, what do we do with these big data after they are marked?

Reply:
I don't know.  I thought they might be skipped in the regular tests, which is why I marked them, but I think we're doing nothing at the moment.